english/bake-cookies/cookies.md

```diff
@@ -2,10 +2,13 @@ Baking is therapeutic for you.  You remember
 learning to bake from your grandmother, or if not your
 grandmother, then off the TV, and how
 relaxing and happy that always makes you.  You
-want to spread the joy (and chocolate chip cookies, or peanut butter, or 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.